src/WebJobs.Extensions.DurableTask/Correlation/StringExtensions.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+
+namespace Microsoft.Azure.WebJobs.Extensions.DurableTask.Correlation
+{
+    /// <summary>
+    /// StringExtensions are the extension method parse string into <see cref="TraceParent"></see> object.
+    /// </summary>
+    public static class StringExtensions
+    {
+        /// <summary>
+        /// Convert a traceParent string to TraceParent object.
+        /// </summary>
+        /// <param name="traceparent">string representations of traceParent.</param>
+        /// <returns>TraceParent object.</returns>
+        public static TraceParent ToTraceParent(this string traceparent)
```

Comment:
I don't know if I love putting an extension method on such a primitive type like `string`. I would almost prefer we just call this static method `TraceParent.FromString()`, and move it under `TraceParent` class.

Reply:
Done